releasenotes/notes/combine-counts-9bd0d6639398af44.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Add an option for the post-processors to combine the counts associated with different parameter
+    values that were submitted in the same POVM sampler PUB.
```

If we do end up with a new how-to guide, we should link to that from here.

---

Let's make sure to link to the new how-to guide here.

We also need to call out the changed behavior of `loc=None`. This should be done in an `upgrade:` section like so:
```suggestion
    Add an option for the post-processors to combine the counts associated with different parameter
    values that were submitted in the same POVM sampler PUB.
upgrade:
  - |
     <explain the new behavior of `loc=None` and how `loc=...` must be used to obtain the old behavior>
```

I don't think we need to deal with this niche case more intricately. We just need to make sure that the next release is a major version bump (which we had planned to do in any case).